src/cudadev/CUDACore/ScopedContext.h

```diff
@@ -227,11 +227,10 @@ namespace cms {
     namespace impl {
       template <typename F>
       void ScopedContextHolderHelper::pushNextTask(F&& f, ContextState const* state) {
-        replaceWaitingTaskHolder(edm::WaitingTaskWithArenaHolder{
-            edm::make_waiting_task_with_holder(std::move(waitingTaskHolder_),
-                                               [state, func = std::forward<F>(f)](edm::WaitingTaskWithArenaHolder h) {
-                                                 func(ScopedContextTask{state, std::move(h)});
-                                               })});
+        replaceWaitingTaskHolder(edm::WaitingTaskWithArenaHolder{edm::make_waiting_task_with_holder(
+            std::move(waitingTaskHolder_), [state, func = std::forward<F>(f)](edm::WaitingTaskWithArenaHolder h) {
```

Comment:
this is just a change in formatting ?

Reply:
Good catch, it indeed seems so. I'll take it away from not creating confusion here (as this PR does not include any other formatting changes).

Reply:
The latest push removed these changes.